code-challenges.test.js

```diff
@@ -26,42 +26,110 @@ const numbersArray2 = [24, 27, 30, 33, 36]
 // --------------------1) Create a function that takes a object as an argument and decides if the number inside it is evenly divisible by three or not.
 
 // a) Create a test with expect statements for each of the variables provided.
-
-const object1 = { number: 15 }
-// Expected output: "15 is divisible by three"
-const object2 = { number: 0 }
-// Expected output: "0 is divisible by three"
-const object3 = { number: -7 }
-// Expected output: "-7 is not divisible by three"
-
+// describe("divisibleBy3", () => {
+//     it("Takes in an object and returns if divisible by three.", () => {
+//       const object1 = { number: 15 }
+//       const object2 = { number: 0 }
+//       const object3 = { number: -7 }
+//       expect(divisibleBy3(object1.number)).toEqual("is divisible by three")
```

Comment:
The prompt is telling you that the function should take an object as an argument, not a value in that argument. Also, make sure to pay attention to what the expected output is. You can see that the number is part of the string that is outputted, so it should also be in your .toEqual.